composer.json

```
@@ -98,6 +98,7 @@
         }
     },
     "scripts": {
-        "fix-cs": "php-cs-fixer fix --config=.php-cs-fixer.dist.php"
+        "fix-cs": "php-cs-fixer fix",
+        "fix-cs-test": "php-cs-fixer fix --ansi --verbose --diff --dry-run"
```

```suggestion
        "lint-cs": "php-cs-fixer fix --ansi --verbose --diff --dry-run"
```
BTW, personally, I'd try to avoid adding more of these shortcuts as they don't bring any added value IMO.

---

The value for me is, that i do not have to remember all the options to run it. I can just run it with a single click. Otherwise this could be added to the readme.

---

IMO, this argument is valid for your private projects. In OS projects, the dependencies must focus on their own responsibilities, relying on other projects where they need to attend different requirements.
What I mean is, the CLI options for php-cs-fixer command are exhaustively documented in [their own docs]( https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/usage.rst#usage).
If we add a new shortcut here for each combination of possible options in the required 3rd party tools, we will end with a big list.

---

> If we add a new shortcut here for each combination of possible options in the required 3rd party tools, we will end with a big list.

Yeah, that's a valid argument, I personally don't use these shortcuts.

I see fine to add something in the Readme file for development though.